pages/page.py

```diff
@@ -32,8 +31,7 @@ def __init__(self, testsetup):
     @property
     def is_the_current_page(self):
         WebDriverWait(self.selenium, 10).until(lambda s: self.selenium.title)
-        Assert.equal(self.selenium.title, self._page_title,
-                     'Expected page title: %s. Actual page title: %s' % (self._page_title, self.selenium.title))
+        assert self._page_title == self.selenium.title
         return True
```

Comment:
That was my first inclination too, but if we do that then the message we get with the assertion failure will just be `False is not True`, whereas if we do the equal assertion here we will see the difference between what we expected and what we got, which will be more useful for debugging.

I know I could have changed all the calls to just `is_the_current_page` without the assertion, as the assertion is being done in this method, but I feel like the intention of the call is more obvious if we are asserting that it is the current page.

What do you think?